web/themes/joinup/joinup_theme.theme

```diff
@@ -992,3 +1018,18 @@ function joinup_theme_preprocess_table__file_formatter_table(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add theme suggestions directly to the comment body field.
+ */
+function joinup_theme_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (strpos($form['#id'], 'views-exposed-form-search-page-') !== FALSE) {
+    $form['#info']['filter-search_api_fulltext']['label'] = '<span class="search-bar__icon icon icon--search"></span>';
```

Comment:
Could you please try to find a way to have an actual textual (even hidden) label? The current way of adding an icon directly without any hint to screen readers might be a problem having [the guidelines](https://www.w3.org/WAI/tutorials/forms/labels/).

Briefly checking the code, there are some visible hints which can be improved:

![search_accessibility](https://user-images.githubusercontent.com/1923476/27865095-81c821e0-6191-11e7-85f6-7b41c18730e0.png)